subjects/devops/shopping/README.md

```diff
@@ -0,0 +1,41 @@
+# shopping
+
+## Instructions
+
+Create a file `shopping.py` with a function `remember_the_apple(shopping_list)`.
+
+This function input is a list of string, like this:
+
+```python
+['tomatoes', 'pastas', 'apple', 'salt']
+```
+
+If the string `'apple'` is not in the list, it should be added. The function return the updated list. If the apple is in the list, simply return it.
+
+If the input list is empty, the returned list should be also empty (don't add the apple in that case).
+
+## Usage
+
+Here is a possible `test.py` to test your functions:
+
+```python
+import shopping
+
+if __name__ == '__main__':
```

Comment:
We don't have a convention, but after I documented a bit I decided not to include that in my examples, since `test.py` is meant to be executed as a script anyway and having this conditionally executed code is useless, since it will make sense only if `test.py` will be imported as a library by another script, which actually makes no sense.

Still, not having a convention fell free to ignore this comment if you think this line should stay, but I'll be of course very happy if you can share your point of view about it ;)

Reply:
I put it "out of habit" even if I haven't coded with python for a while - I will remove it also not to confuse the student with something that might look weird to someone that is not familiar with the language.